go/genqlient.yaml

```diff
@@ -0,0 +1,8 @@
+schema: ../../backend/public-graph/graph/schema.graphqls
+operations:
+- internal/gql/operations.graphql
+generated: internal/gql/generated.go
+
+bindings:
+  Any:
+    type: interface{}
```

Comment:
This may change if that ends up inconvenient implementing the sampler. This is used for attribute comparison and could be any JSON type.